src/components/TooltipProvider/TooltipWrapper.tsx

```diff
@@ -27,25 +26,23 @@ const TooltipWrapper = ({
     }
   }, [])
 
-  return React.cloneElement(children, {
-    ref: (ref: HTMLElement) => {
-      anchorRef.current = ref
-      if (forwardRef) {
-        // eslint-disable-next-line no-param-reassign
-        forwardRef.current = ref
-      }
-    },
-    'data-tooltip-place': place,
-    'data-tooltip-content': content,
-    'data-tooltip-html': html,
-    'data-tooltip-variant': variant,
-    'data-tooltip-offset': offset,
-    'data-tooltip-wrapper': wrapper,
-    'data-tooltip-events': events,
-    'data-tooltip-position-strategy': positionStrategy,
-    'data-tooltip-delay-show': delayShow,
-    'data-tooltip-delay-hide': delayHide,
-  })
+  return (
+    <span
+      ref={anchorRef}
+      data-tooltip-place={place}
+      data-tooltip-content={content}
+      data-tooltip-html={html}
+      data-tooltip-variant={variant}
+      data-tooltip-offset={offset}
+      data-tooltip-wrapper={wrapper}
+      data-tooltip-events={events}
+      data-tooltip-position-strategy={positionStrategy}
+      data-tooltip-delay-show={delayShow}
+      data-tooltip-delay-hide={delayHide}
+    >
+      {children}
+    </span>
+  )
```

Comment:
After some testing, I've realized `cloneElement()` doesn't work super well when passing functional components as children to the wrapper.

Using an actual DOM element (`span` seems ok) didn't seem to cause any issues.